elasticapm/utils/cgroup.py

```diff
@@ -100,6 +100,6 @@ def parse_cgroups(filehandle):
             pod_id = kubepods_match.group(1)
             if not pod_id:
                 pod_id = kubepods_match.group(2)
```

Comment:
should only be replacing in this case, when we're matching the systemd pattern (kubepods.slice, etc.)

Reply:
That's what I did as well. Not sure if this is a real problem though, meaning- not sure which has more chances of requiring us to reiterate in the future. But for the sake of consistency...

Reply:
Good call. I was unsure if we can apply the replacement to pod IDs in general or only if systemd is involved. Better to be safe and consistent tho :+1: